.pipelines/templates/template-job-deploy-azure-env.yml

```diff
@@ -10,116 +10,54 @@ parameters:
-              deployerDirectory: $(System.ArtifactsDirectory)/deployer/drop
-              configFileName: ${{ parameters.configFileName }}
-              location: ${{ location }}
-              azureDevOpsJSONSPN: ${{ parameters.azureDevOpsJSONSPN }}
-              fullDeploy: ${{ parameters.fullDeploy }}
-  - job: RP_E2E_${{ location }}
-    dependsOn: Deploy_${{ location }}
-    variables:
-    - template: ../vars.yml
-    timeoutInMinutes: 120
-    pool:
-      name: ARO-CI
-    steps:
-    - template: ./template-prod-e2e-steps.yml
-      parameters:
-        location: ${{ location }}
-        subscription: ${{ parameters.e2eSubscription }}
-        azureDevOpsE2EJSONSPN: ${{ parameters.azureDevOpsE2EJSONSPN }}
-        aroVersionStorageAccount: ${{ parameters.aroVersionStorageAccount }}
-        rpMode: ${{ parameters.rpMode }}
-  - job: Billing_E2E_${{ location }}_Wait
-    dependsOn: RP_E2E_${{ location }}
-    timeoutInMinutes: 400
-    pool: server
-    steps:
-    - task: Delay@1
-      inputs:
-        delayForMinutes: '360'
-  - job: ${{ location }}_TriggerBillingBuild
-    displayName: Trigger Billing E2E pipeline
-    dependsOn: Billing_E2E_${{ location }}_Wait
-    steps:
-    - script: |
-        # Pass variables between tasks: https://medium.com/microsoftazure/how-to-pass-variables-in-azure-pipelines-yaml-tasks-5c81c5d31763
-        echo "##vso[task.setvariable variable=REGION]${{ location }}"
-        CLUSTER="v4-e2e-V$BUILD_BUILDID-${{ location }}"
-        echo "##vso[task.setvariable variable=CLUSTER]$CLUSTER"
-        CLUSTER_RESOURCEGROUP="v4-e2e-V$BUILD_BUILDID-${{ location }}"
-        echo "##vso[task.setvariable variable=CLUSTER_RESOURCEGROUP]$CLUSTER_RESOURCEGROUP"
-        echo "E2E Cluster Resource Group Name:" $CLUSTER_RESOURCEGROUP
-        echo "E2E Cluster Name:" $CLUSTER
-      displayName: Pass variables into next Task
-    - task: TriggerBuild@3
-      inputs:
-        definitionIsInCurrentTeamProject: true
-        buildDefinition: ${{ parameters.billingE2EPipelineName }}
-        queueBuildForUserThatTriggeredBuild: true
-        ignoreSslCertificateErrors: false
-        useSameSourceVersion: false
-        useCustomSourceVersion: false
-        useSameBranch: false
-        branchToUse: ${{ parameters.billingE2EBranchName }}
-        waitForQueuedBuildsToFinish: true
-        storeInEnvironmentVariable: false
-        buildParameters: CLUSTER_RESOURCEGROUP:$(CLUSTER_RESOURCEGROUP), CLUSTER:$(CLUSTER),
-          REGION:$(REGION)
-        authenticationMethod: OAuth Token
-        password: $(System.AccessToken)
-        enableBuildInQueueCondition: false
-        dependentOnSuccessfulBuildCondition: true
-        dependentOnFailedBuildCondition: true
-        checkbuildsoncurrentbranch: false
-        failTaskIfConditionsAreNotFulfilled: true
+          - checkout: |
+              git://AzureRedHatOpenShift/RP-Config@${{ parameters.rpConfigCommit }}
```

Comment:
This does not work with:
![image](https://user-images.githubusercontent.com/3225744/104595968-3a07a100-566b-11eb-9542-0da82db97420.png)
 
There is some issues how VSTS handles checkouts with inline syntax and commit hash'es

Reply:
I have a feeling it does it via ref not commit resolution ;/